common/src/main/java/com/microsoft/identity/common/internal/result/MsalBrokerResultAdapter.java

```diff
@@ -338,6 +338,12 @@ public Bundle bundleFromBaseException(@NonNull final BaseException exception,
         }
 
         if (exception instanceof IntuneAppProtectionPolicyRequiredException) {
+            // Record presence of exception in telemetry
+            SpanExtension.current().setAttribute(
+                    AttributeName.is_mam_flow.name(),
+                    Boolean.toString(true)
```

Comment:
Is there a reason we need to convert to String? What are we doing for all the existing boolean? I suggest we follow the same pattern

Reply:
Seems like string conversion for booleans is mainly used outside of telemetry purposes, i've switched back to just passing the boolean itself